src/components/AdminLink.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+
+import type { RouteParams } from "../types";
+
+import React, { Component } from "react";
+import { Link } from "react-router";
+
+import url from "../url";
+
+
+type Props = {
+  name: string,
+  params: RouteParams,
+  children: any,
+};
+
+export default class AdminLink extends Component {
```

Comment:
Creating a small test for this one should be cheap and valuable enough methinks.